websites/D/DiscordLink/presence.ts

```diff
@@ -0,0 +1,39 @@
+const presence = new Presence({
+  clientId: "627875524298932254"
+});
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo-dlm"
+  };
+
+  presenceData.startTimestamp = Math.floor(Date.now() / 1000);
+
+  if (window.location.pathname.endsWith("/")) {
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "Home Page";
+  } else if (window.location.pathname.endsWith("/add")) {
+    presenceData.details = "Adding a server";
+  } else if (window.location.pathname.endsWith("/view")) {
+    presenceData.details = "Viewing a server";
+    presenceData.state = document.querySelector(
+      "#bigytxt > span"
+    ).textContent;
+  } else if (window.location.pathname.startsWith("/edit")) {
+    presenceData.details = "Editing a server";
+    presenceData.state = document.querySelector(
+      "#bigytxt > span"
+    ).textContent;
+  } else if (window.location.pathname.startsWith("/tags")) {
+    presenceData.details = "Viewing tags list";
+  } else if (window.location.pathname.includes("/store")) {
+    presenceData.details = "Viewing the Store";
+  }
```

Comment:
It would be easier to use this:
`const page = document.location.pathname;` and then you can simply use `if (path === "YOURPAGE")`

Reply:
Ok, can we approve it like this and im going to work and test stuff then ill update the presence with your reccomendations and new dynamic page headers such as tags and member counts etc.